src/js/features/clinPhen/makeGetPhenopacket.thunk.ts

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { phenopacketUrl } from '@/constants/configConstants';
+import type { RootState } from '@/store';
+import { printAPIError } from '@/utils/error.util';
+import { authorizedRequestConfig } from '@/utils/requests';
+import type { Phenopacket } from '@/types/clinPhen/phenopacket';
+import { RequestStatus } from '@/types/requests';
+import { getIsAuthenticated } from 'bento-auth-js';
+
+export const makeGetPhenopacketData = createAsyncThunk<
+  Phenopacket,
+  string,
+  {
+    state: RootState;
+    rejectValue: string;
+  }
+>(
+  'query/makeGetPhenopacketData',
+  (id, { rejectWithValue, getState }) => {
+    return axios
+      .get(`${phenopacketUrl}/${id}`, authorizedRequestConfig(getState()))
+      .then((res) => res.data)
+      .catch(printAPIError(rejectWithValue));
+  },
+  {
+    condition(id, { getState }) {
+      const isAuthenticated = getIsAuthenticated(getState().auth.idTokenContents);
+      const state = getState().clinPhen;
+
+      const cond =
+        !!isAuthenticated &&
```

Comment:
this logic isn't correct - you need to check the `query:data` permission access, not whether we're authenticated, since auth COULD let anonymous users see phenopackets.